code/_onclick/item_attack.dm

```diff
@@ -160,8 +160,29 @@
 
 /// The equivalent of [/obj/item/proc/attack] but for alternate attacks, AKA right clicking
 /obj/item/proc/attack_alt(mob/living/victim, mob/living/user, params)
+	if(sharpness & (SHARP_POINTY | SHARP_EDGED))
+		sharpnessStabCheck(victim, user, params)
+		return
```

Comment:
You still need to return something here--the attack chain will `CRASH` otherwise. You want `ALT_ATTACK_CONTINUE_CHAIN`.

Reply:
aye, I wasnt sure what to use there so I relied on crashing 😅